config/initializers/sidekiq.rb

```diff
@@ -20,3 +21,19 @@
   require 'sidekiq/testing'
   Sidekiq::Testing.inline!
 end
+
+if ENV['REDIS_HOST'].present? && ENV['REDIS_PASSWORD'].present?
```

Comment:
The problem being if we need to change this URL at any point then we are required to make infrastructure changes and have this updated by the infra team. We have all the required fields supplied to us so we might aswell utilise this power within our code rather than outsource to our infra.